examples/iris/iris.rb

```diff
@@ -31,10 +31,10 @@ def fwd(x)
 optimizer.setup(model)
 
 iris = Datasets::Iris.new
-x = iris.each.map {|r| r.each.to_a[0..3]}
+x = iris.to_table.to_h.values[0..3].transpose
```

Comment:
`Table#to_h` always parses dataset.
It's better that reuse `Table` object for performance:

```ruby
iris_table = iris.to_table
x = [iris_table[:sepal_length], iris_table[:sepal_width], iris_table[:petal_length], iris_table[:width]].transpose
```

It may be better that we add `Table#fetch_values` like `Hash#fetch_values`.

Reply:
> ```Table#to_h``` always parses dataset.
> It's better that reuse ```Table``` object for performance:

Thank you for review.
I fixed it to reuse the Table object.

Reply:
I've released red-datasets 0.0.5 with `Table#fetch_values`.
Now, we can write as the following:

```ruby
x = iris_table.fetch_values(:sepal_length, :sepal_width, :petal_length, :width).transpose
```

Reply:
>  Now, we can write as the following:

Thanks!
 I fixed it to use Table#fetch_values.